docker-app/qfieldcloud/settings.py

```diff
@@ -224,8 +224,7 @@
 
 TIME_ZONE = os.environ.get("QFIELDCLOUD_DEFAULT_TIME_ZONE") or "Europe/Zurich"
 
-USE_I18N = False
-
+USE_I18N = True
```

Comment:
I just remembered why `USE_I18N` was disabled. When enabled, all errors are returned in the user's language sent by the browser. Which gets a strange mixture between English and the user's language.

We want it disabled for app.qfield.cloud for now, so I can imagine it is going to be a new envvar.